src/screens/NetworkSettings/CustomNetworkSettingsScreen.js

```diff
@@ -217,6 +224,15 @@ export const CustomNetworkSettingsScreen = ({ navigation }) => {
           value={formModel.explorerServiceUrl}
         />
 
+        <SimpleInput
+          containerStyle={styles.input}
+          label={t`Tx Mining Service URL`}
```

Comment:
I think I prefer the label as 'Transaction Mining Service URL'. We don't use the contraction 'tx' anywhere in the app, so to keep consistency I think we should continue using 'Transaction' here.

What do you think?

Reply:
I agree! It is clearer, despite `Tx` be ubiquitous in the industry.

Reply:
Done at: https://github.com/HathorNetwork/hathor-wallet-mobile/pull/390/commits/4abc76376f843e901f160581ce8e36bdabf78898